multipledispatch/tests/test_conflict.py

```diff
@@ -60,3 +61,46 @@ def test_ordering():
 
 def test_type_mro():
     assert super_signature([[object], [type]]) == [type]
+
+
+def test_supercedes_variadic():
+    assert supercedes((Variadic[B],), (Variadic[A],))
+    assert supercedes((B, Variadic[A]), (Variadic[A],))
+    assert supercedes((Variadic[A],), (Variadic[(A, C)],))
+    assert supercedes((A, B, Variadic[C]), (Variadic[object],))
+    assert supercedes((A, Variadic[B]), (Variadic[A],))
+    assert supercedes(tuple([]), (Variadic[A],))
+    assert supercedes((A, A, A), (A, Variadic[A]))
+    assert not supercedes((Variadic[A],), (Variadic[B],))
```

Comment:
actually, it looks like these are just flipped arguments from the positive tests, should we write a little helper like:

```python
def assert_supersedes(a, b):
    assert supercedes(a, b)
    assert not supercedes(b, a)
```

and then write each pair once

Reply:
Sounds good.